### ISSUE TITLE
Status and future of BME280 as of 38-rc16

### ISSUE THREAD

**Issue by sumerland** [nettigo/namf#35]
I read the release notes of rc16 but do not fully understand. Does that mean that support of BME280 will be removed from the firmware? The data is still showing up on the LCD and the website, but what is sent to api-rrd.madavi is obviously broken (maps.sensor.community seems to be fine).

Please keep support for the BME280.

![image](https://user-images.githubusercontent.com/64929473/128194644-c9f24619-437e-4e70-b046-618bb1192542.png)

![image](https://user-images.githubusercontent.com/64929473/128194713-e81dda15-2b42-48cf-b6b3-a3131cf24df9.png)

![image](https://user-images.githubusercontent.com/64929473/128194801-f1009ee2-6ca6-4f1e-b09e-2331429f1aff.png)

**Comment by netmaniac** [comment]
Sorry for confusion. 

BME will be supported, but due to lack of modules on the market we had to change our kits. New kits will include SHT31 for temp & humidity and BMP180 for pressure. So we have added support for BMP180 to fw. But due to my mistake data is being sent even when no BMP180 is enabled.

I have just released rc17 with fixed that problem with BMP and also BME is moved into new scheduler.

For BME difference is that temp/press/humidity are 5 times measured during measurement period and average is being sent not just result of single measurement before sending data.
